### DISCUSSION TITLE
get all router as the same yarn build

### DISCUSSION THREAD

**Discussion by synasapmob** [vercel/next.js#64440] [App Router]
### Question

possible to catch all router of **app?**, I just wanna show list route UI to make something
when **yarn build** all route will show in terminal and I followed code in a repository, but isn't my expected

correctly:
![Screen Shot 2024-04-13 at 16 07 10](https://github.com/vercel/next.js/assets/42969529/e70af860-bf57-4a2a-8757-4ff0ed899032)

wrong:
![Screen Shot 2024-04-13 at 16 26 03](https://github.com/vercel/next.js/assets/42969529/abbd35e3-a08e-4e16-9e3a-8dfc62c6b53b)

**Comment by croissain** [comment]
I had the same issue

**Reply by synasapmob** [reply]
crying..
